internal/versions/versions.go

```diff
@@ -103,6 +106,29 @@ func (h *handler) V2ListSupportedOpenshiftVersions(ctx context.Context, params o
 			supportedArchs = []string{*releaseImage.CPUArchitecture}
 		}
 
+		// (MGMT-11859) We are filtering out multiarch release images so that they are available
+		//              only for customers allowed to use them. This is in order to be able to
+		//              expose them in OCP pre-4.13 without making them generally available.
+		if len(supportedArchs) > 1 {
+			hasMultiarchAuthorization := false
+			checkedForMultiarchAuthorization := false
+
+			if !checkedForMultiarchAuthorization {
+				checkedForMultiarchAuthorization = true
+				if err := auth.ValidateAccessToMultiarch(ctx, h.authzHandler); err != nil {
+					if strings.Contains(err.Error(), auth.NoAccessToMultiarch) {
+						continue
+					} else {
+						return common.GenerateErrorResponder(err)
+					}
+				}
+				hasMultiarchAuthorization = true
+			}
+			if !hasMultiarchAuthorization {
```

Comment:
Given the if statement in line 118 + the assignment in 125, this is always `false`

Reply:
We have now merged the prerequisite to master, so subsystem test should tell us if this part works. I think after evacuating the initial values to the top of the function it's okay now